stable/postgresql/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: postgresql
-version: 0.17.0
+version: 0.17.1
```

Comment:
since this is a new feature, I would say to bump to `0.18.0`

Reply:
Done